sql/hive-thriftserver/src/test/scala/org/apache/spark/sql/hive/thriftserver/HiveThriftServer2Suite.scala

```diff
@@ -70,6 +73,34 @@ class HiveThriftServer2Suite extends FunSuite with Logging {
     port
   }
 
+  def withThriftUIAndJDBC(
```

Comment:
This method duplicates a lot code in `withJdbcStatement`.  It should be easy to eliminate them.